roles/openshift_logging/filter_plugins/openshift_logging.py

```diff
@@ -102,6 +102,38 @@ def serviceaccount_namespace(qualified_sa, default=None):
     return seg[-1]
 
 
+def build_protected_line_string(protected_lines):
```

Comment:
Can you give an example of where/how this is used?

Reply:
What I mean is - does `line` need to be regex escaped?  bash shell escaped?

Reply:
I added comments for the wrong method... this can likely be scrapped. We aren't using it, it's been functionally replaced by `account_for_whitelist` in `lib/logging_patch.py`